README.md

```diff
@@ -250,3 +251,22 @@ const ItemsList = (props) => {
 
 export default connectApiData(ItemsList);
 ```
+
+## Configure with Redux-persist: 
+```js
+    
```

Comment:
Voornaamste doel is om `afterRehydrate()` toe te lichten (sorry, heb ik niet duidelijk omschreven). 
Daarnaast is het belangrijk dat configureApiData() gedispatchd wordt nadat rehydration klaar is. Daar is de callback van persistStore voor. Met uitleg dat als je dit niet doet en je maakt config changes, dat deze dan mogelijk weer overschreven worden door de rehydrate.